src/components/picker/picker.tsx

```diff
@@ -203,7 +203,12 @@ export class Picker {
     @Watch('value')
     protected onChangeValue(newValue = [], oldValue = []) {
         this.chips = this.createChips(this.value);
-        if (newValue.length <= oldValue.length) {
+
+        if (!this.multiple) {
```

Comment:
This won't work, we would still have the bug for pickers that have `multiple` set to true.

We have to fix that we do not set focus on the element when the value is changed. We want to do it when selecting an item from the dropdown I guess, but not _everytime_ the value is changed :dizzy_face:

Reply:
Let's see if you're happy with this solution :D
- The element won't get focus when the chipSet isn't defined (from start)
- The new card component in the webclient we've never used messed some stuff up and therefore I'm removing it